datastore/admin/datastore_admin_test.go

```diff
@@ -45,4 +51,15 @@ func TestCreate(t *testing.T) {
 	if got.IndexId != want {
 		t.Fatalf("Unexpected indexID: got %v, want %v", got.IndexId, want)
 	}
+	// Create bucket for Export/Import entities.
+	bucketName := tc.ProjectID + "-storage-bucket-test"
```

Comment:
Perhaps this should have a timestamp to prevent namespace clashes?

Reply:
Thank you for review! 
Oh, I used [this bucket creation](https://github.com/GoogleCloudPlatform/golang-samples/blob/56f12b11eec1006b65de0b8270ddaba42d3dfab8/storage/buckets/buckets_test.go#L32) as an example. So we need to update it and other's samples too?

Reply:
Gotcha-- maybe @tbpg can chime in here on the best approach? He would know better.

Reply:
Appears to be a unique suffix and the call to `CleanBucket` should prevent staleness issues. So, this seems OK to me.